glean_parser/lint.py

```diff
@@ -281,6 +311,7 @@ def check_expired_metric(
     "BUG_NUMBER": (check_bug_number, CheckType.error),
     "BASELINE_PING": (check_valid_in_baseline, CheckType.error),
     "MISSPELLED_PING": (check_misspelled_pings, CheckType.error),
+    "REDUNDANT_PING": (check_redundant_ping, CheckType.error),
```

Comment:
This will need to go further below, around line 322, as we only want that check running on `pings`, not `metrics`.

Reply:
I see, now I placed the
"REDUNDANT_PING": (check_redundant_ping, CheckType.error),

under the PING_CHECKS: Dict[..]